src/app/modules/user/actions.ts

```diff
@@ -0,0 +1,51 @@
+import { Action } from 'redux';
+import { ThunkAction } from '../../store';
+import {
+  getRuns, 
+  getCourtrooms 
+} from '../courthouse/action';
+import {getSheriffList} from '../sheriffs/actions';
+import { getAssignments, getAssignmentDuties, getAssignmentDutyDetails } from '../assignments/actions';
+import { getShifts } from '../shifts/actions';
+
+// The following gives us type-safe redux actions
+// see https://medium.com/@dhruvrajvanshi/some-tips-on-type-safety-with-redux-98588a85604c
+// Todo: Would be great to make this more generic and factor it out into infrastructure, leaving as is for now
+
+type IActionMap = {
+  'USER_UPDATE_CURRENT_COURTHOUSE': string;
```

Comment:
maybe `string|undefined` allowing us to clear it?

Reply:
fixed by update to selector